python/lsst/ap/association/association.py

```diff
@@ -98,7 +121,16 @@ def run(self,
                 nUpdatedDiaObjects=0,
                 nUnassociatedDiaObjects=0)
 
-        matchResult = self.associate_sources(diaObjects, diaSources)
+        if self.config.doTrailedSourceFilter:
+            diaTrailedResult = self.trailedSourceFilter.run(diaSources, exposure_time)
+            matchResult = self.associate_sources(diaObjects, diaTrailedResult.diaSources)
+
+            self.log.warning("%i DIASources exceed maxTrailLength, dropping "
```

Comment:
Why a warning? Nothing went wrong, since the intent was to remove sources. `log.info` is probably fine.

Reply:
I saw that the other log.info that drops a source was a warning. However, you are right since dropping the source makes it work as intended so it should be a log.info.